joomla/joomla-cms
# [5.2] Mail Templates language file loading not using the MailsHelper::loadTranslationFiles

### Summary of Changes

Load the template language files via the MailsHelper over an manuall code which not handles it correctly for example for plugins.

### Testing Instructions

1. Install this extension: https://github.com/zero-24/plg_task_extensionupdates/releases
2. check the mail templates (system -> Mail Templates -> Extensions Update Notification)
3. notice that the mail template is not translated
4. apply this patch
5. notice the templates are now translated
6. check other mail templates
7. confirm they are also still translated

### Actual result BEFORE applying this Pull Request

On the detail view an mail template shipped with an plugin is not translated
![image](https://github.com/joomla/joomla-cms/assets/2596554/508ff12e-39ff-45b1-99b1-58c9326cd3a5)

### Expected result AFTER applying this Pull Request

Once that patch is applied the issue is solved
![image](https://github.com/joomla/joomla-cms/assets/2596554/2140347c-d11b-4782-aa80-d817e29f8746)

### Link to documentations
Please select:
